.github/pull_request_template.md

```diff
@@ -0,0 +1,96 @@
+<!--
+You've got a Pull Request you want to submit? Awesome!
+This PR template is here to help ensure you're setup for success:
+  please fill it out to help ensure that your PR is complete and ready for approval.
+-->
+
+**JIRA Ticket:**
+[SOMEPROJECT-42](https://jira.cms.gov/browse/SOMEPROJECT-42)
+
+**User Story or Bug Summary:**
+<!-- Please copy-paste the brief user story or bug description that this PR is intended to address. -->
+
+
+### What Does This PR Do?
+
+<!--
+Add detailed discussion of changes here
+This is likely a summary, or the complete contents, of your commit messages.
+-->
+
+
+### What Should Reviewers Watch For?
+
+If you're reviewing this PR, please check these things, in particular:
+
+* TODO
+
+<!--
+Add some items to the list above, or remove the entire section if it doesn't apply for some reason.
+
+Common items include:
+* Is this likely to address the goals expressed in the user story?
+* Are any additional documentation updates needed?
+* Are there any unhandled and/or untested edge cases you can think of?
+* Is user input properly sanitized & handled?
+* Does this make any backwards-incompatible changes that might break end user clients?
+* Can you find any bugs if you run the code locally and test it manually?
+-->
+
+
+### What Security Implications Does This PR Have?
+
+Submitters should complete the following questionnaire:
+
+* If the answer to any of the questions below is **Yes**, then here's a link to the associated Security Impact Assessment: N/A.
```

Comment:
There's more nuance here.  In BCDA, we have a security checklist that helps us determine whether an SIA is needed (it asks these questions and more); linking to the security checklist would be an appropriate substitute if a SIA was _not_ needed.  

In DPC, we follow a similar but less formal process.   Our understanding is that context matters; for instance, a non-security-related dependency that is limited to development or CI/CD should not trigger a SIA.  If we're not sure, we do reach out to @StewGoin, but almost always in a more private forum than the public PR review.

This may just be something each team has to tweak to fit their security process.

Reply:
@dhgreene If we can at all manage it, I'd prefer to avoid having teams customize their templates, as that will make change management a mess -- it'll be a lot harder to roll out changes to this template if everyone's customized it.

Can you suggest any wording changes/additions here that would make it a better fit for BCDA & DPC?

Also worth noting that this language was clear with @StewGoin ahead of this PR going out. That's not to say it can't be improved! Merely pointing out that it's at least _sufficient_ from his perspective.

Reply:
```suggestion
* If the answer to any of the questions below is **Yes**, then here's a link to the associated Security Impact Assessment (SIA) (or security checklist documenting why a SIA is not needed): N/A.
```

Reply:
@dhgreene What about this, instead?
```suggestion
* If the answer to any of the questions below is **Yes**, then here's a link to the associated Security Impact Assessment (SIA), security checklist, or other similar document in Confluence: N/A.
```